modules/nw-ingress-controller-configuration-proxy-protocol.adoc

```diff
@@ -6,9 +6,18 @@
 [id="nw-ingress-controller-configuration-proxy-protocol_{context}"]
 = Configuring the PROXY protocol for an Ingress Controller
 
-A cluster administrator can configure https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt[the PROXY protocol] when an Ingress Controller uses either the `HostNetwork` or `NodePortService` endpoint publishing strategy types. The PROXY protocol enables the load balancer to preserve the original client addresses for connections that the Ingress Controller receives. The original client addresses are useful for logging, filtering, and injecting HTTP headers. In the default configuration, the connections that the Ingress Controller receives only contain the source address that is associated with the load balancer.
+A cluster administrator can configure https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt[the PROXY protocol] when an Ingress Controller uses either the `HostNetwork` or `NodePortService` endpoint publishing strategy types. The PROXY protocol enables the load balancer to preserve the original client addresses for connections that the Ingress Controller receives. The original client addresses are useful for logging, filtering, and injecting HTTP headers. In the default configuration, the connections that the Ingress Controller receives only contain the source IP address that is associated with the load balancer.
 
-This feature is not supported in cloud deployments. This restriction is because when {product-title} runs in a cloud platform, and an IngressController specifies that a service load balancer should be used, the Ingress Operator configures the load balancer service and enables the PROXY protocol based on the platform requirement for preserving source addresses.
+[IMPORTANT]
+====
+For a passthrough route configuration, client pods in an {product-title} cluster cannot access their original client source IP address from within the cluster. Ensure that you configure Ingress access logging for your Ingress Controller so that you can view what source IP addresses client pods can access from within your cluster.
+
+For re-encrypt and edge routes, you can set the `X-Forwarded-For` value in the `httpHeaders` of the `ingresscontrollers.operator.openshift.io` resource to allow the client source IP address to reach client pods in a cluster.
```

Comment:
OpenShift router sets the `forwarded` and `x-forwarded-for` headers itself; in the general case, there should be no need to configure `httpHeaders` actions to set these headers.
```suggestion
For re-encrypt and edge routes, {product-title} router sets the `Forwarded` and `X-Forwarded-For` headers to enable servers to check the client source IP address.
```

Reply:
:heavy_check_mark: